.env.development

```diff
@@ -10,7 +10,7 @@ NEXT_PUBLIC_NODE_2 = "https://bsc-dataseed1.defibit.io"
 # 10+ nodes balanced in each region, global
 NEXT_PUBLIC_NODE_3 = "https://bsc-dataseed.binance.org"
 
-NEXT_PUBLIC_NODE_PRODUCTION = "https://nodes.pancakeswap.com"
+NEXT_PUBLIC_NODE_PRODUCTION = ""
```

Comment:
Maybe remove the line completely?

Reply:
yup I thought we have to override. Thanks